### PR TITLE
Fixed getMasterRequest() deprecations

### PR DESCRIPTION
**Pull request by brusch**

### CODE REVIEW

**Comment by blankse** [comment]
@brusch getMainRequest doesn't exists in symfony/http-foundation 5.2.0
Is part of 5.3.0. So you must bump composer, right?

**Comment by brusch** [comment]
@blankse you're totally right - good catch! Thanks!

**Comment by blankse** [comment]
@brusch I wonder why the phpstan test lowest also get 5.3. Seems to be impossible to get 5.2.0 now.

Edit: symfony/security-bundle is 5.3 and require  symfony/http-foundation 5.3. So it can't be 5.2

**Comment by brusch** [comment]
@blankse true, but should be fixed anyway 😉

File: lib/Http/RequestHelper.php [hunk 1]
```diff
@@ -82,17 +82,32 @@ public function getRequest(Request $request = null)
      */
     public function hasMasterRequest(): bool
```
Comment (jdreesen) [inline lib/Http/RequestHelper.php:83]: I'd say this should be deprecated and replaced by `hasMainRequest()`, too.

Comment (brusch) [reply lib/Http/RequestHelper.php:83]: Absolutely 😊 See b3fd6ee1370c764478daaf1a50fe66632ec6067d

**Pull request merged**